content/swift/concepts/dictionaries/terms/values/values.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.values'
+Description: '.value is used to loop through a collection of values within a dictionary.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+  - 'Computer Science'
+Tags:
+  - 'Dictionary'
+  - 'Properties'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.Values`** method is used to loop through a stored collection of values within a key-value dictionary. 
```

Comment:
```suggestion
The **`.values`** property in Swift is used to access a collection of all the values stored in a key-value dictionary.
```

Reply:
We don't necessarily loop, but if needed, it can be looped using iteration methods like `for-in`.